### PR TITLE
Fixes #11189 - copy and load fixtures from tmpdir

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: test/katello_test_helper.rb [hunk 1]
```diff
@@ -56,9 +56,13 @@ module FixtureTestCase
     self.pre_loaded_fixtures = true
 
     Katello::FixturesSupport.set_fixture_classes(self)
-    load_fixtures
-    self.fixture_path = "#{Katello::Engine.root}/test/fixtures/models"
+
+    fixture_path = Dir.mktmpdir("katello_fixtures")
+    FileUtils.cp(Dir.glob("#{Katello::Engine.root}/test/fixtures/models/*"), fixture_path)
+    FileUtils.cp(Dir.glob("#{Rails.root}/test/fixtures/*"), fixture_path)
+    self.fixture_path = fixture_path
```
Comment (ehelms) [inline test/katello_test_helper.rb]: Does this need to be set after the copy versus as the first line in this set of statements?

Comment (jlsherrill) [inline test/katello_test_helper.rb]: oh no, i was actually setting a 'local' variable which probably wasn't so local!

Comment (jlsherrill) [inline test/katello_test_helper.rb]: and not actually needed

Comment (jlsherrill) [inline test/katello_test_helper.rb]: updated

**Comment by ehelms** [comment]
ACK - hopefully this fixes the issue! Might be nice to extract this to Foreman itself once more than just us needs to use fixtures from multiple plugins.

**Pull request merged**